package.json

```diff
@@ -9,6 +9,7 @@
     "test:system": "tap test/system --timeout=600",
     "test:integration": "DEPLOYMENT_TYPE=YAML TEST_PLATFORM=kind CREATE_CLUSTER=true tap test/integration/kubernetes.test.ts --timeout=900",
     "test:integration:kind": "DEPLOYMENT_TYPE=YAML TEST_PLATFORM=kind CREATE_CLUSTER=true tap test/integration/kubernetes.test.ts --timeout=900",
+    "test:integration:kind:helm": "DEPLOYMENT_TYPE=Helm TEST_PLATFORM=kind CREATE_CLUSTER=true tap test/integration/kubernetes.test.ts --timeout=900",
```

Comment:
how about adding 
```"test:integration:kind:yaml"```
which would be a copy-paste of 
test:integration:kind":
and at some point delete test:integration:kind AND test:integration?

Reply:
I thought about it but preferred to do it in a future PR, wanted to scope the changes only to Helm, but definitely the intention to use :yalm and remove the other two 🙏